Sources/Subs-Db-postgresql.php

```diff
@@ -499,7 +499,12 @@ function smf_db_query($identifier, $db_string, $db_values = array(), $connection
 
 /**
  * affected_rows
- * @param resource $connection
+ *
+ * !!! Not sure what this is supposed to do.
```

Comment:
Return the affected rows of a query.
see  http://php.net/manual/de/function.pg-affected-rows.php